packages/contracts/contracts/dummy/upgradeCheck/README.md

```diff
@@ -26,25 +26,25 @@ User ---- tx ---> Proxy ----------> Implementation_v0
 - Update deploy script.
 
 - Ensure V0 factory contract is deployed.
-    - Command: `yarn run goerli-deploy-program-factory`
+    - Command: `yarn run deploy-program-factory goerli`
```

Comment:
Change `goerli` to a placeholder like `<network>`?

Reply:
^ Oh that's purely an example where we are attempting to deploy goerli.
The actual readme -> state `<network>`